.github/workflows/test.yaml

```diff
@@ -464,6 +465,7 @@ jobs:
       DATABASE_DO_PG_BOUNCER_URL: ${{ secrets.DATABASE_DO_PG_BOUNCER_URL }}
       HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
       DATABASE_HEROKU_PGBOUNCER_URL: ${{ secrets.DATABASE_HEROKU_PGBOUNCER_URL }}
+      DATABASE_URL_DB_MSSQL: ${{ secrets.DATABASE_URL_DB_MSSQL }}
```

Comment:
What database is this using? I can see no service or docker definition.

Reply:
AWS RDS MSSQL (in 1password already).

Reply:
Do we usually test external databases for the basic database tests?
If not, we should switch this to Docker as well to not depend on an external service here.
If testing that service is relevant, we can do that as well of course if there is value in it.

Reply:
Almost (if not all) the tests in this repo so far are on an external Postgres RDS, I just did the same for consistency.

Reply:
Ah, we don't actually have tests for pure mysql and postgres - that is done in the integration tests.
All good then - but maybe rename it to include `aws-rds` so it is clear what is being tested.

Reply:
Renamed to `mssql-aws-rds`, as the test is green.

Reply:
(firebase-functions failed but that is ofc unrelated to this PR)